Source/Softswitch/inc/softswitch.h

```diff
@@ -1,46 +1,52 @@
 #ifndef _SOFTSWITCH_H_
 #define _SOFTSWITCH_H_ 
 
-#include <cstring>
 #include "poets_msg.h"
 
-#define NUM_SUP_BUFS 4
 #define MAX_LOG_MSG_BUFS 4
 
-class handler_log_msg
-{
-    public:
-    handler_log_msg();
-    uint32_t      pack_fmtstr(const char* &);
-    void          send_msg();
-      
-#ifndef TRIVIAL_LOG_HANDLER
-    // argument expansion and buffer packing
-    template<typename... U> inline void pack_log_args(U&&... args) {uint32_t _a[] = {0, (prepack_log_arg(args),0)...};}
-    // argument extractor
-    template<typename P> inline void prepack_log_arg(P arg) {pack_log_arg<P>(sizeof(P),&arg);};
-    // does the actual work
-    template<typename V> void pack_log_arg(uint32_t, V*);
+#define P_DEFAULT_LOG_LEVEL     2
+
+#ifndef P_LOG_LEVEL
+#define P_LOG_LEVEL P_DEFAULT_LOG_LEVEL
 #endif
-     
-    P_Sup_Msg_t*  log_msg[MAX_LOG_MSG_BUFS];   // buffer for sequenced parts of the log message
-    uint32_t      msg_len;                     // running message length
-    uint32_t      seq;                         // and running sequence count
-};
+
+//------------------------------------------------------------------------------
+// Tinsel Slot Allocations.
+// Slot 0 is used to send "normal" messages.
+// Slot 1 is used for supervisor messages (e.g. instrumentation)
```

Comment:
Messages -> Packets

Also, supervisor messages and instrumentation messages are fundamentally different. Perhaps this should say "Slot 1 is used for supervisor packets and instrumentation packets.

Reply:
This will be an issue throughout - I wrote the softswitch from the Tinsel viewpoint, where what we call Packets are Messages...

I could go through and zap all occurences of message/msg, but there are 115 in the doccumentation alone...

Reply:
> Also, supervisor messages and instrumentation messages are fundamentally different. Perhaps this should say "Slot 1 is used for supervisor packets and instrumentation packets.

Good spot, should really be "control" messages

Reply:
From our discussion yesterday (IIRC), we decided on

 - "Messages" for MPI traffic

 - "Packets" for backend traffic

 - "Tinsel Messages" for backend traffic when talking about concepts that are Tinsel-specific.

We also decided to change packet classes (Mostly just `P_Msg_t`) to packet equivalents (best efforts).

Reply:
Message > Packet change made across the softswitch and relevant bits of the mothership in e0fee08bc0056ec8d3e2bbdc8495efa71c1c6459